tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -95,7 +95,10 @@ Define {
 	BT_PB_DAI_PIPELINE_SRC "copier.host.9.1"
 	BT_PB_PIPELINE_STREAM_NAME "dai-copier.SSP.10.1"
 	GOOGLE_RTC_AEC_SUPPORT		0
+	# assign core for AEC LL pipelines
 	GOOGLE_AEC_CORE_ID		0
```

Comment:
Do we need to run AEC as LL?

Reply:
@marcinszkudlinski  I think this is for the rest of the pipeline on which AEC is.

Reply:
yes, for example, as a test we can run echo ref related pipelines on core 2, and put RTC_AEC module alone on core 1:

<img src="https://github.com/thesofproject/sof/assets/19287706/3c1548bd-eeca-4322-a596-fb80c8362057" width="400">

Reply:
ok, so comment is a little confusing. We should update it to say AEC is using DP and rest of pipeline is LL and both are on different cores.

Reply:
Ack, will update commit message as well as the PR description, thanks,